.gitmodules

```diff
@@ -33,3 +33,6 @@
 [submodule "src/isa-l"]
 	path = src/isa-l
 	url = https://github.com/ceph/isa-l
+[submodule "src/blkin"]
+	path = src/blkin
+	url = https://github.com/linuxbox2/blkin
```

Comment:
https://github.com/ceph/blkin and https://github.com/ceph/babeltrace-zipkin are available now